modules/promotion/src/Entity/Promotion.php

```diff
@@ -428,6 +443,15 @@ public function available(OrderInterface $order) {
         return FALSE;
       }
     }
+    if ($usage_limit_per_client = $this->getUsageLimitPerClient()) {
+      /** @var \Drupal\commerce_promotion\PromotionUsageInterface $usage */
+      $usage = \Drupal::service('commerce_promotion.usage');
```

Comment:
Do not call services as static but use dependency injection

Reply:
Forget about it, i miss we are in  ContentEntityBase

Reply:
ok